bin/picca_export.py

```diff
@@ -240,6 +241,44 @@ def main(cmdargs):
         'R-parallel', 'R-transverse', 'Redshift', 'Correlation',
         'Covariance matrix', 'Distortion matrix', 'Number of pairs'
     ]
+
+    # Check if we need blinding and apply it
+    if 'BLIND' in data_name or blinding != 'none':
+        if blinding != 'corr_yshift':
+            print('Only strategy E called "corr_yshift" is allowed for now.'
+                  ' This will be applied automatilly.')
+
+        # Figure out what which correlation it is
+        if len(xi) == 50:
+            if 'lyb' in args.data or 'Lyb' in args.data:
+                blind_corr = 'lyaxlyb'
+            else:
+                blind_corr = 'lyaxlya'
+        elif len(xi) == 100:
+            assert 'qso' in args.data
+            if 'lyb' in args.data or 'Lyb' in args.data:
+                blind_corr = 'qsoxlyb'
+            else:
+                blind_corr = 'qsoxlya'
+        else:
+            raise ValueError('Unknown binning. Cannot blind. Please raise an issue'
+                             ' or contact picca developers.')
+
+        # Read the blinding file and get the right template
+        blinding_path = ('/global/cfs/projectdirs/desi/users/acuceu/notebooks/'
```

Comment:
Consider renaming it to `blinding_filename`. Add a check that makes sure the file exists and complain otherwise
```
if not os.path.isfile(blinding_path):
    raise ValueError("Missing blinding file. Make sure you are running at"
                                " NERSC or contact picca developers")
```

Reply:
Resolved by commit #728c397